tei_config.rb

```diff
@@ -44,8 +44,8 @@
 
 ms_contents = 'tei:msContents'
 to_field 'cho_description', extract_tei("#{ms_desc}/#{ms_contents}/tei:summary")
-to_field 'cho_language', main_language, transform(&:downcase), translation_map('not_found', 'languages', 'marc_languages')
-to_field 'cho_language', other_languages, transform(&:downcase), translation_map('not_found', 'languages', 'marc_languages')
+to_field 'cho_language', transform(&:downcase), translation_map('not_found', 'languages', 'marc_languages')
```

Comment:
This line and the next line currently do nothing, because they're no longer invoking the methods defined in `Macros::TEI`.

Reply:
Ok, I think I fixed it this time